src/app/modules/did-book/components/did-book-list/did-book-list.component.ts

```diff
@@ -34,4 +36,14 @@ export class DidBookListComponent {
   remove(id: string) {
     this.delete.emit(id);
   }
+
+  qrCodeData(element: DidBookRecord): QrCodeData {
+    return {
+      type: ScanType.User,
+      did: element.did,
+      additionalData: {
+        label: element.label,
+      },
```

Comment:
I think maybe it would be better to keep it more generic in the sense that we can expect every QR code to have a `type` and some `data`, but not necessarily have a `did`, I think
```suggestion
      type: ScanType.User,
      data: {
        did: element.did,
        label: element.label,
      },
```

Reply:
Yeah, good idea. I will adjust code. Thanks!